common/models/container.js

```diff
@@ -92,7 +92,8 @@ module.exports = function(Container) {
                     if(row[singleOrder.storeName] > 0){
                       singleOrder.items.push({
                         sku : row["sku"],
-                        fulfilledQuantity : row[singleOrder.storeName]
+                        fulfilledQuantity : row[singleOrder.storeName],
+                        orderQuantity : row[singleOrder.storeName]
```

Comment:
since this is inside a generic method, i want to make sure that having `orderQuantity` be the same as `fulfilledQuantity` makes sense for all business scenarios .... does it? If you've thought this through then just ignore this comment.